packages/labs/testing/src/lib/fixtures/csrFixture.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {render} from 'lit';
+import type {LitElement, TemplateResult} from 'lit';
+import type {FixtureOption} from './fixtureOption.js';
+
+/**
+ * Renders the provided lit-html template with a Lit element client-side.
+ *
+ * @param {TemplateResult} template - lit-html template. Must contain a single
```

Comment:
(Same question as above -- does it really need to be a top-level custom element?)